src/components/inputs/select-input/select-input.js

```diff
@@ -242,22 +243,26 @@ export class SelectInput extends React.Component {
                   }
                 : undefined
             }
-            onChange={selectedOptions =>
-              // selectedOptions is either an array, or a single option
+            onChange={selectedOptions => {
+              // selectedOptions is either an array, or a single option, or null
               // depending on whether we're in multi-mode or not (isMulti)
```

Comment:
before if you cleared your multi select, the value was empty array. now it's null

Reply:
I guess another option would be to continue sending empty array, that way we won't break anything in the MC 🤔

Reply:
yeah I guess otherwise this would be a breaking change. 😱

Reply:
I would prefer to "wrap" the breaking change. I would assume that forms, mapping into and out of them, would break otherwise.

Reply:
yeah, I'm currently doing this. Just investigating (console logging 😆) the old output of our selects when values are cleared, and going through (and adding tests 😄 ) to make sure new one matches. I did it for select-input already, need to do it for the other 3

Reply:
alright done